fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedViewPartitionNameRef.java

```diff
@@ -0,0 +1,86 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.catalog;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class MaterializedViewPartitionNameRef implements Writable {
```

Comment:
what is the difference between this class and MaterializedViewPartitionRef?

Reply:
this used for persist partition name map, `MaterializedViewPartitionRef` used for  persist version map